### PR TITLE
added work option to boston

### PR DESCRIPTION
**Pull request by vumapathy**

### CODE REVIEW

**Comment by vumapathy** [comment]
Please review this, @casey-collab

**Comment by pmallory** [comment]
@casey-collab Can you take a look at this PR?

File: english/boston/boston.md [hunk 1]
```diff
@@ -2,4 +2,7 @@
 
 "This city is the life! Though now that I'm here, I think I'll take a vacation..."
 
+...or on the second thought, I'll put on a coat, stop being a sissy, and get to work!
+
 [Go on vacation](../licked-cat/vacation.md)
+[Get to work](../gtd/gtd.md)
```
Comment (casey-collab) [inline english/boston/boston.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by vumapathy** [comment]
@casey-collab updated!

**Comment by casey-collab** [comment]
Looks great! :fire:

**Pull request merged**